content/blog/2016/2016-06-16-parallel-test-executor-plugin.adoc

```diff
@@ -68,6 +68,14 @@ void runTests(def args) {
 void mvn(def args) { /* ... */ }
 ----
 
+[NOTE]
+====
+This pipeline expects to be run as part of a multibranch pipeline.
+To run it stand-alone, replace the `checkout scm` step with
+`checkout([$class: 'GitSCM', userRemoteConfigs: [[url: 'https://github.com/jenkinsci/git-plugin.git']]])`.
```

Comment:
IMHO this is more succinctly stated with `git 'https://github.com/jenkinsci/git-plugin.git'` instead of the verbose `checkout` step